demisto_sdk/commands/content_graph/tests/create_content_graph_test.py

```diff
@@ -144,14 +154,16 @@ def mock_classifier(name: str = "SampleClassifier"):
 def mock_playbook(
     name: str = "SamplePlaybook",
     marketplaces: List[MarketplaceVersions] = [MarketplaceVersions.XSOAR],
+    fromversion="5.0.0",
```

Comment:
Better use a constant (I think we have a `GENERAL_MINIMUM_FROM_VERSION` or something, if we don't have one for playbooks.

Reply:
this is a just a mock.
I think we need to handle this when we convert this mocks to the TestSuite